applications/asset_tracker_v2/doc/sensor_module.rst

```diff
@@ -60,12 +60,11 @@ The threshold is set in one of the following two ways:
 
 Both events contain an accelerometer threshold value ``accelerometer_threshold`` in m/s2, present in the event structure.
 
-Motion detection in the module is enabled and disabled in turn to control the number of :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` events that are sent out by the sensor module.
-This functionality acts as flow control.
-It prevents the sensor module from sending events to the rest of the application if the device is accelerating over the threshold for extended periods of time.
+Motion detection in the module is enabled and disabled according to the mode setting of the device.
+Motion detection will be enabled in passive mode and disabled in active mode.
 
-The application module controls this behavior through the :c:enum:`APP_EVT_ACTIVITY_DETECTION_ENABLE` and :c:enum:`APP_EVT_ACTIVITY_DETECTION_DISABLE` events.
-The sensor module will only send out a :c:enum:`SENSOR_EVT_MOVEMENT_DATA_READY` event if it detects movement and activity detection is enabled.
+The sensor module will send out a :c:enum:`SENSOR_EVT_MOVEMENT_ACTIVITY_DETECTED` event if it detects movement.
+Similarly, :c:enum:`SENSOR_EVT_MOVEMENT_INACTIVITY_DETECTED` is sent out if there was no movement for a while.
 
```

Comment:
Remember to update the `sensor_module.rst` docs in https://github.com/nrfconnect/sdk-nrf/pull/8020 after this is merged to include:
- Documentation on how activity/inactivity thresholds as well as inactivity time is set. Both compile time and real-time. The initial section in `Motion detection` can probably be expanded to encapsulate more configs.
- Refer to the inactivity threshold instead of `for a while` so that users know exactly what parameter the should manipulate to extend the inactivity time.